x/evm/keeper/migrate.go

```diff
@@ -15,7 +15,8 @@ const uaxlAsset = "uaxl"
 // GetMigrationHandler returns the handler that performs in-place store migrations from v0.17 to v0.18. The
 // migration includes:
 // - delete pending chains from the base keeper
-// - migrate contracts bytecode (CRUCIAL AND DO NOT DELETE)
+// - add VotingGracePeriod param for all evm chains
+// - migrate contracts bytecode (CRUCIAL AND DO NOT DELETE) for all evm chains
```

Comment:
This isn't done. Is this comment meant to be gone, or is the migration still to come?

Reply:
see line 38